src/TagField.php

```diff
@@ -275,7 +275,7 @@ protected function getOptions($onlySelected = false)
 
         if ($values) {
             if (is_array($values)) {
-                $values = DataList::create($dataClass)->filter($titleField, $values);
+                $values = $source->filter($titleField, $values); // DataList::create($dataClass)->filter($titleField, $values);
```

Comment:
Seems sensible. Would you mind removing the commented out code here, and updating the phpcs violation on line 393?

Reply:
@robbieaverill fixed those.